frontend/src/pages/org/settings/settings.ts

```diff
@@ -426,19 +426,95 @@ export class OrgSettings extends TailwindElement {
+            >
+              <div
+                class="col-start-2 flex items-baseline justify-between gap-2"
+              >
+                ${msg("Admin")}
+                <span class="text-xs text-gray-500">
+                  ${msg("Manage org and billing settings")}
+                </span>
+              </div>
+              <sl-details
+                @sl-hide=${this.stopProp}
+                @sl-after-hide=${this.stopProp}
+                class="col-span-2 part-[content]:p-2 part-[header]:p-2 part-[content]:pt-0 group-attr-[aria-checked=true]:part-[base]:border-primary/50 group-attr-[aria-checked=true]:part-[summary-icon]:text-primary-700"
+              >
+                <span slot="summary" class="text-xs">Permissions</span>
+                <ul class="ms-4 list-disc text-xs text-gray-500">
+                  <li class="text-warning">${msg("Manage org members")}</li>
+                  ${this.appState.settings?.billingEnabled &&
+                  html`<li class="text-warning">
+                      ${msg("Edit billing details")}
+                    </li>
+                    <li class="text-warning">
+                      ${msg("Update or cancel subscription")}
+                    </li>`}
+                  <li>${msg("Create crawl workflows")}</li>
+                  <li>${msg("Create browser profiles")}</li>
+                  <li>${msg("Upload archived items")}</li>
+                  <li>${msg("Run QA analysis")}</li>
+                  <li>${msg("Rate and review archived items")}</li>
+                  <li>${msg("Create, edit, and share collections")}</li>
+                </ul>
+              </sl-details>
             </sl-radio>
-            <sl-radio value=${AccessCode.crawler}>
-              ${msg("Crawler — Can create crawls")}
+            <sl-radio
+              value=${AccessCode.crawler}
+              class="group rounded-md border border-neutral-300 p-2 transition-colors hover:border-neutral-400 attr-[aria-checked=true]:border-primary attr-[aria-checked=true]:bg-primary-50 part-[control]:col-start-1 part-[label]:col-start-1 part-[control]:col-end-2 part-[label]:col-end-3 part-[control]:row-start-1 part-[label]:row-start-1 part-[control]:row-end-2 part-[label]:row-end-3 part-[label]:ml-0 part-[base]:grid part-[label]:grid part-[label]:flex-auto part-[base]:grid-cols-[auto_minmax(0,1fr)] part-[label]:grid-cols-subgrid part-[base]:gap-x-1 part-[label]:gap-y-2"
+            >
+              <div
+                class="col-start-2 flex items-baseline justify-between gap-2"
+              >
+                ${msg("Crawler")}
+                <span class="text-xs text-gray-500">
+                  ${msg("Create, evaluate, and curate archives")}
```

Comment:
To my knowledge we're trying to avoid the term "archive"? (cc @Shrinks99)
```suggestion
                  ${msg("Create, evaluate, and curate archived items")}
```

Reply:
Correct, though we do use it as a verb to describe the act of "archiving" content.